less/forum.less

```
@@ -91,3 +91,8 @@
   position: absolute;
   .Button--color(@tooltip-color, @tooltip-bg);
 }
+@media screen and (max-width: 550px) {
```

I recommend using the phone breakpoint variable built into Flarum.
```suggestion
@media screen and (max-width: @screen-phone-max) {
```